roadmap/implementors-guide/src/node/backing/candidate-selection.md

```diff
@@ -12,7 +12,7 @@ In the event that a parablock candidate proves invalid, this subsystem will rece
 
 ## Protocol
 
-Input: None
+Input: Parablock candidates and associated Proofs of Validity from collators
```

Comment:
I've imagined that this subsystem will internally handle network logic for receiving these.

The other protocol sections refer to Input/Output in terms of messaging between subsystems, which this doesn't fall into the category of

Reply:
OK, I've updated it.